### PR TITLE
Shared networkmanager API usage

### PR DESCRIPTION
**Pull request by pvuorela**
Depends on
https://github.com/sailfishos/libconnman-qt/pull/27

### CODE REVIEW

**Review by LaakkonenJussi — APPROVED** [review]
LGTM

File: connectionagentplugin/declarativeconnectionagent.cpp [hunk 1]
```diff
@@ -31,8 +31,8 @@ DeclarativeConnectionAgent::DeclarativeConnectionAgent(QObject *parent)
       connManagerInterface(nullptr)
 {
     connectiondWatcher = new QDBusServiceWatcher(CONND_SERVICE,QDBusConnection::sessionBus(),
```
Comment (spiiroin) [inline connectionagentplugin/declarativeconnectionagent.cpp]: as there are "X,Y" -> "X,[space]Y" fixes made elsewhere... one more to fix here

File: test/auto/tst_connectionagent/tst_connectionagent.cpp [hunk 2]
```diff
@@ -46,39 +45,20 @@ void Tst_connectionagent::tst_onErrorReported()
     QSignalSpy spy(&agent, SIGNAL(errorReported(QString,QString)));
     agent.onErrorReported("test_path","Test error");
```
Comment (spiiroin) [inline test/auto/tst_connectionagent/tst_connectionagent.cpp]: another spaceless comma

**Review by spiiroin — APPROVED** [review]
ok with or without further comma-space fixes

File: connectionagentplugin/declarativeconnectionagent.cpp (hunk 1 above)
Comment (pvuorela) [reply connectionagentplugin/declarativeconnectionagent.cpp]: Sure.

**Pull request merged**